scripts/flash/do-initial-report.js

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env node
+
+"use strict";
+
+var chalk = require("chalk");
+
+function doInitialReport(augur, marketID, payoutNumerators, invalid, auth, callback) {
+  augur.api.Market.doInitialReport({
+    meta: auth,
+    tx: { to: marketID  },
+    _payoutNumerators: payoutNumerators,
+    _invalid: invalid,
+    onSent: function (result) {
+      console.log(chalk.yellow.dim("Sent:"), chalk.yellow(JSON.stringify(result)));
+      console.log(chalk.yellow.dim("Waiting for reply ...."));
+    },
+    onSuccess: function (result) {
+      console.log(chalk.green.dim("Success:"), chalk.green(JSON.stringify(result)));
+      callback(result);
```

Comment:
`result` is passed as the first (and only) argument to the callback, but in scripts/flash/designated-report.js's invocation of this function (line 26) the callback's argument is defined to be an error, and upon receiving an error the script automatically exits.

To avoid this sort of thing, I strongly suggest using Node-style callbacks, with the error always as the first argument, and the result as the second argument.